CONTRIBUTING.md

```diff
@@ -69,6 +69,16 @@ x = 3
 print('Number in var x is: {x}'.format(x=x))
 ```
 
+## Update the version
+
+Before creating the pull request, please update the version in `drip/__inti__.py` following the next rules:
+
+- If you are doing a fix or a very small feature, then bump the third digit.
+- If you are adding a new feature, then bump the second digit.
+- If you are doing a breaking changes or a very big feature, then bump the first digit.
```

Comment:
I think we need to define a better way to establish the version releases(in my opinion semantic versioning fits for this project).
Here's a couple of references:
- [Maintaining Open Source Projects: Versioning and Releasing](https://thoughtbot.com/blog/maintaining-open-source-projects-versioning#time-based-releases)
- [Semantic versioning](https://semver.org/)
@rootstrap/python-maintainers